docs/testing.md

```diff
@@ -9,6 +9,7 @@ Code verification includes lint, code formatting, boilerplate check etc.
 ```bash
 make install.tools
 ```
+***Note:*** The `GOPATH` needs to be set to run the make target.
```

Comment:
maybe something like "Some make actions use the user's GOPATH and will otherwise not work when it is not set, while other make actions override it by setting it to a temporary directory for release build and testing purposes."

Reply:
Reads better. Updated.